packages/module/patternfly-docs/content/examples/EmptyDetailsView/EmptyDetailsViewBasic.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { EmptyDetailsView } from '@patternfly/ai-infra-ui-components';
+import { Button } from '@patternfly/react-core';
+import clusterImage from './empty-state-cluster-storage.svg';
+
+export const EmptyDetailsViewBasic: React.FunctionComponent = () => (
+  <EmptyDetailsView
+    title="Start by adding cluster storage"
+    description="Cluster storage saves your project’s data on a selected cluster. You can optionally connect cluster storage to a workbench."
+    iconImage={clusterImage}
+    imageSize="240px"
```

Comment:
Can we change `240px` to `320px`

Also, this size is different than the size shown in [/EmptyDetailsView/EmptyDetailsView.tsx](https://github.com/patternfly/AI-infra-ui-components/pull/65/files#diff-44250e9a015464c3a624c3887392326fbc2268687d43678915577af8a2de2070) below, which is using 320px. 

We have new [guidelines for empty states](https://docs.google.com/document/d/1xq9DjJDh4-P0Lj4hcctqmJrfgZneYgsvdUOqp1hr7Lw/edit?tab=t.0) that recommend using 320px for illustrations. It would be ideal if our examples in the preview use the values we are recommending.